sherpa/models/regrid.py

```diff
@@ -502,13 +409,16 @@ class ModelDomainRegridder1D():
     instead it creates an instance that is used to evaluate
```

Comment:
An instance of what? Of itself? Of an Evaluation space?

Reply:
I didn't change this text in this PR so I'm reluctant to add more changes if I can help it here.

If you are interested, the example provides some context, but I think if we were to fix this then we'd have to fix the 2D case to, and then end up explaining the whole file, which is just not going to happen any time soon.

Reply:
understood.